Bottom: f5d17045527c1ab882a16ac9644622fcbc3e4d46
Top:    5e995325f36c62694bb28e2a3e17eb906a83047c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-27 11:18:24 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index df90d8c..0a8461e 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -61,7 +61,7 @@ ghid_make_gc (void)
   return rv;
 }
 
-void
+static void
 ghid_draw_grid ()
 {
   static GLfloat *points = 0;
